### DISCUSSION TITLE
How to host Flet app

### DISCUSSION THREAD

**Discussion by jvan00** [flet-dev/flet#4364] [Q&A]
Dear everyone I want to ask how we can host and connecting to domains and servers

**Comment by tanmay-bhatgare** [comment]
there are many ways to host flet app like,
### [Static](https://flet.dev/docs/publish/web/static-website)
- Cloudflare
- Github Page (best option for static ones)

### [Dynamic](https://flet.dev/docs/publish/web/dynamic-website/hosting)
- fly.io (need some purchases)
- github pages (free to host)
- replit (not recommended tho)

**Reply by Muddassir-Farooq-official** [reply]
GitHub pages doesn't support dynamic website hosting

**Reply by tanmay-bhatgare** [reply]
Yeah my bad. Idk why I point it out. Apologies!

**Comment by vihutuo** [comment]
Try railway, Its one click deployment  from github directly. You maybe able to host 4-5 low traffic apps for $5 a month
